### ISSUE TITLE
PIP does not come preinstalled; Installing manually fails

### ISSUE THREAD

**Issue by TheRealKeto** [ProcursusTeam/Procursus#40]
**Description**
The current version of Python that the bootstrap hosts does not automatically install ``pip``. A quick fix I've tried was to "manually" install ``pip`` by running ``python -m ensurepip``, but that also didn't work. 

I'm sure with some changes either ``pip`` can be installed automatically, or be installed manually by running the command above. I have provided a error log below for reference.

```
Ketos-iPod:~ root# python -m ensurepip
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.8/ensurepip/__main__.py", line 5, in <module>
    sys.exit(ensurepip._main())
  File "/usr/lib/python3.8/ensurepip/__init__.py", line 210, in _main
    return _bootstrap(
  File "/usr/lib/python3.8/ensurepip/__init__.py", line 129, in _bootstrap
    return _run_pip(args + [p[0] for p in _PROJECTS], additional_paths)
  File "/usr/lib/python3.8/ensurepip/__init__.py", line 32, in _run_pip
    runpy.run_module("pip", run_name="__main__", alter_sys=True)
  File "/usr/lib/python3.8/runpy.py", line 207, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.8/runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/__main__.py", line 16, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/__init__.py", line 40, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/cli/autocompletion.py", line 8, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/cli/main_parser.py", line 11, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/commands/__init__.py", line 6, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/commands/completion.py", line 6, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/cli/base_command.py", line 21, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_internal/download.py", line 14, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_vendor/requests/__init__.py", line 114, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_vendor/requests/utils.py", line 26, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_vendor/requests/_internal_utils.py", line 11, in <module>
  File "<frozen zipimport>", line 259, in load_module
  File "/tmp/tmpxl5kqymk/pip-19.2.3-py2.py3-none-any.whl/pip/_vendor/requests/compat.py", line 62, in <module>
  File "/usr/lib/python3.8/urllib/request.py", line 2619, in <module>
    from _scproxy import _get_proxy_settings, _get_proxies
ModuleNotFoundError: No module named '_scproxy'
Ketos-iPod:~ root#  
```

**Device Information**
Device: iPod 6
Version: iOS 12.4.1
Bootstrap: Odyssey + Procursus
Jailbreak: checkra1n

**Comment by M1staAwesome** [comment]
should be fixed with e5c2bad

**Comment by Diatrus** [comment]
Fixed, per @M1staAwesome
